code-studio/src/js/code-studio.js

```diff
@@ -2,7 +2,56 @@
  * @file Replacement for application.js.erb in dashboard as much as possible.
  *       Code added here will be loaded for (almost) every page in dashboard,
  *       so use sparingly!
+ *
+ * Note: This is included _after_ application.js.erb during our transition period,
+ * so when moving things preserve the include order as much as possible.
  */
 'use strict';
+/* global Sounds */
 
 require('./videos');
+window.React = require('react');
+require('./components/*.jsx', {mode: 'expand'});
```

Comment:
The [`require-globify` transform](https://github.com/capaj/require-globify) lets us import whole directories like this, which is the closest analogue I found to the sprockets directive that was including our React components before.